exercises/practice/hello-world/.meta/config.json

```diff
@@ -1,6 +1,23 @@
 {
   "blurb": "The classical introductory exercise. Just say \"Hello, World!\"",
-  "authors": [],
+  "authors": [
+    "michaelem"
+  ],
+  "contributors": [
+    "behrtam",
+    "cmccandless",
+    "Dog",
+    "hebertjulio",
+    "ikhadykin",
+    "kytrinyx",
+    "N-Parsons",
+    "NobbZ",
```

Comment:
I can not remember having contributed to this exercise nor do I find myself in the history of relevant exercise files, please remove me from this list